systemtests/src/test/java/io/enmasse/systemtest/clients/BasicClientTest.java

```diff
@@ -0,0 +1,200 @@
+package io.enmasse.systemtest.clients;
+
+import io.enmasse.systemtest.AddressSpace;
+import io.enmasse.systemtest.AddressSpaceType;
+import io.enmasse.systemtest.Destination;
+import io.enmasse.systemtest.MultiTenantTestBase;
+import io.enmasse.systemtest.executor.client.AbstractClient;
+import io.enmasse.systemtest.executor.client.Argument;
+import io.enmasse.systemtest.executor.client.ArgumentMap;
+import io.enmasse.systemtest.executor.client.rhea.RheaClientReceiver;
+import io.enmasse.systemtest.executor.client.rhea.RheaClientSender;
+import org.junit.Test;
+
+import java.util.Random;
+import java.util.concurrent.Future;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class BasicClientTest extends MultiTenantTestBase {
+
+    @Test
+    public void basicMessageTest() throws Exception {
+        AddressSpace addressSpace = new AddressSpace("brokered-send-receive",
+                "brokered-send-receive",
+                AddressSpaceType.BROKERED);
+        createAddressSpace(addressSpace, "none");
+        Destination dest = Destination.queue("message-basic");
+        setAddresses(addressSpace, dest);
+
+        ArgumentMap arguments = new ArgumentMap();
+        arguments.put(Argument.USERNAME, "test");
+        arguments.put(Argument.PASSWORD, "test");
+        arguments.put(Argument.BROKER, getRouteEndpoint(addressSpace).toString());
+        arguments.put(Argument.ADDRESS, dest.getAddress());
+        arguments.put(Argument.LOG_MESSAGES, "json");
+        arguments.put(Argument.COUNT, "10");
+        arguments.put(Argument.CONN_SSL, "true");
+
+        AbstractClient sender  = new RheaClientSender();
```

Comment:
Clients(with arguments) should be prepared in setUp method before each test and should be added into the List of clients. Only code of test should iterate through this list. TearDown method should clear this list. It was the original reason why we did AbstractClient, executor, etc...